src/content/docs/fr/tutorial/1-setup/4.mdx

```diff
@@ -0,0 +1,70 @@
+
+1. Connectez-vous à GitHub.com dans un navigateur et cliquez sur le <kbd>+</kbd> en haut à droite de l'écran pour créer un nouveau référentiel.
+
+2. Choisissez un nom pour votre référentiel. Il n'a pas besoin d'avoir le même nom que le dossier de votre projet.
+
+3. Vous verrez différentes options, mais vous n'avez pas besoin de modifier les paramètres par défaut. Faites défiler vers le bas et cliquez sur le bouton <kbd>Créer le référentiel</kbd>.
+
+4. Vous verrez diverses étapes de configuration, mais vous n'aurez pas besoin de les utiliser. Notez l'URL de votre référentiel. Vous pouvez maintenant quitter cette page sans rien faire.
+
+## Validez votre code local sur GitHub
+
+Dans la dernière section, vous avez apporté une modification au contenu de votre page. Cela signifie que les fichiers de votre projet ont été modifiés, et VS Code devrait afficher un nombre en haut de l'icône du menu "Source". Cet onglet Source est l'endroit où vous irez régulièrement pour mettre à jour vos fichiers sur GitHub.
+
+1. Cliquez sur l'onglet Contrôle de source dans VS Code pour voir la liste des fichiers modifiés. Si vous voyez un message vous indiquant que vous devez installer `git`, suivez les instructions fournies, puis rechargez VS Code.
+
+2. Cliquez sur le menu des "3 points" <kbd>•••</kbd> au-dessus du message de validation et choisissez <kbd>Remote</kbd> > <kbd>Ajouter une télécommande</kbd>.
+
+3. Sélectionnez <kbd>Ajouter une télécommande depuis GitHub</kbd>. Si nécessaire, suivez les étapes d'authentification, puis revenez à VS Code et répétez cette action.
+
+4. Vous devriez voir une liste de tous vos référentiels sur GitHub. Choisissez celui que vous avez créé pour ce projet. Si vous ne voyez pas votre projet, collez directement son URL GitHub. Vous pouvez également être invité à donner un nom local à ce référentiel. Vous pouvez choisir n'importe quel nom qui vous convient.
+
+5. En haut du volet de menu, il y aura un endroit pour saisir un **message de validation** (description de vos modifications de fichier). Tapez "commit initial" et appuyez sur le bouton <kbd>Valider</kbd> pour valider ces modifications.
+
+7. Vous pouvez voir un message vous indiquant que vous n'avez pas de validations "mises en scène" et vous demandant si vous voulez les mettre en scène. Cliquez sur <kbd>Toujours</kbd> et continuez.
+
+8. Enfin, la liste des fichiers modifiés devrait être remplacée par un bouton <kbd>Publier</kbd>. Cliquez dessus pour envoyer vos modifications validées sur GitHub.
+
+### Voir votre projet sur GitHub
+
+Pour vérifier que votre projet est stocké avec succès sur GitHub, rendez-vous sur GitHub.com et recherchez sous votre compte une liste de vos référentiels. Choisissez le nouveau que vous avez créé, et vérifiez qu'il contient les fichiers de votre projet Astro.
```

Comment:
```suggestion
Pour vérifier que votre projet est stocké avec succès sur GitHub, rendez-vous sur GitHub.com et recherchez sous votre compte une liste de vos dépôts. Choisissez le nouveau que vous avez créé, et vérifiez qu'il contient les fichiers de votre projet Astro.
```
I think we could replace `le nouveau` by `celui` as this reflects the point well enough and is easy to read.
```suggestion
Pour vérifier que votre projet est stocké avec succès sur GitHub, rendez-vous sur GitHub.com et recherchez sous votre compte une liste de vos dépôts. Choisissez celui que vous avez créé, et vérifiez qu'il contient les fichiers de votre projet Astro.
```